doc/modules/changes/20191019_jrobey

```diff
@@ -0,0 +1,6 @@
+Fixed: The volume of fluid composition advection algorithm incorrectly computed
+the boundry fluxes on inflow boundries with fixed composition. This has been
```

Comment:
boundry -> boundary

Also to make it easier to understand please reorder to: "The volume of fluid composition advection algorithm  computed
the boundary fluxes on inflow boundries with fixed composition incorrectly."